main/store/migrations/index.js

```diff
@@ -407,7 +407,66 @@ const migrations = {
     })
 
     return initial
-  }
+  },
+  21: initial => {
+    // add sepolia network information
+    if (!initial.main.networkPresets.ethereum[11155111]) {
+      initial.main.networkPresets.ethereum[11155111] = {
+        infura: 'infuraSepolia'
+      }
+    }
+
+    if (!initial.main.networks.ethereum[11155111]) {
+      initial.main.networks.ethereum[11155111] = {
+        id: 11155111,
+        type: 'ethereum',
+        layer: 'testnet',
+        symbol: 'ETH',
+        name: 'Sepolia',
+        explorer: 'https://sepolia.etherscan.io',
+        gas: {
+          price: {
+            selected: 'standard',
+            levels: { slow: '', standard: '', fast: '', asap: '', custom: '' }
+          }
+        },
+        connection: {
+          primary: { on: true, current: 'infura', status: 'loading', connected: false, type: '', network: '', custom: '' },
+          secondary: { on: false, current: 'custom', status: 'loading', connected: false, type: '', network: '', custom: '' }
+        },
+        on: false
+      }
+    }
+
+    if (!initial.main.networksMeta.ethereum[11155111]) {
+      initial.main.networksMeta.ethereum[11155111] = {
+        gas: {
+          fees: {},
+          price: {
+            selected: 'standard',
+            levels: { slow: '', standard: '', fast: '', asap: '', custom: '' }
+          }
+        }
+      }
+    }
+    
+    // reset and switch off goerli if any removed RPCs are active
```

Comment:
can this comment be updated to something that explains what's going on here for future devs? maybe something like "we removed support for X RPCs so reset the user's connection to goerli to defaults if they were previously connecting to those RPCs"